### ISSUE TITLE
Installing on fresh Raspbian Stretch Pi3

### ISSUE THREAD

**Issue by shawnipoo** [resin-io/resin-wifi-connect#124]
How can I go about installing this on a fresh install of Raspbian Stretch on a Pi3?
Thank you!

**Comment by majorz** [comment]
@shawnipoo, we prepared an install script for the upcoming version (still not merged in master). The script won't work with the current version without some modifications though:
https://github.com/resin-io/resin-wifi-connect/blob/116-raspbian-install-script/scripts/resin-wifi-connect-on-raspbian.sh

It basically disables dhcpcd and enables NetworkManager as a networking manager.

**Comment by shawnipoo** [comment]
Thank you. What type of modifications are required? Is there a document showing said modifications?

Finally, when will the upcoming version be released?

**Comment by majorz** [comment]
Actually let me extract the necessary steps for disabling dhcpcd and enabling NetworkManager:

```
sudo apt-get install -y -d network-manager
sudo systemctl stop dhcpcd
sudo systemctl disable dhcpcd
sudo apt-get install -y network-manager
sudo systemctl enable NetworkManager
sudo systemctl start NetworkManager
```

After this you can download the current release from https://github.com/resin-io/resin-wifi-connect/releases/latest and then run the start script in the main folder.

The upcoming version is coming very soon now. It should be really released at the beginning of next week unless something unpredictable happens.

**Comment by shawnipoo** [comment]
Thank you for that. The start script fails because it references app.js which is not found. The src folder only contains app.coffee...

**Comment by majorz** [comment]
@shawnipoo, yesterday we released v3.0.0, which is now distributed as a binary. My mistake above was that the previous version was published as source only, and this is why app.js was not found in the archive. 

You may use the shell script above for installation of v3.0.0 without modifications now. I will do another round of testing and will merge the installation script. We will follow up soon with a blog post about installing the application on Raspbian Stretch as well.

**Comment by majorz** [comment]
@shawnipoo, the Raspbian Stretch install script is committed now with v3.0.4. It is located in the `scripts` folder in case you want to give it a try. Closing the issue now.
